Wp 473

kwlodarska opened the pull request:
Mobile modification of test bed application. I created media query for styles and additional javascript file only for mobile look. Unfortunately I had to modify a little orginal html code (i.e. remove tables) but then I style it to look like before for PC version.

Jira task:
http://jira.webinos.org/browse/WP-473

ghost commented:
Hi Katarzyna,

Please update the git commit messages according to the instructions on the dev portal (more meaningful commit messages, reference jira issue).
Thanks!

kwlodarska committed:
Test bed adjustment for mobile view

I created css media query for mobile view, separate javascript file
for mobile view functions and I changed some html code when needed
(but I style it to look same as before for PC page).

Jira issue: WP-473

habibvirji commented on webinos/test/client/client.html:
```diff
@@ -62,7 +63,10 @@
           $('#enableLogin').html("<a href='#login' id='login'> Connect to Your PZH </a>");
           $("#login").click(function(){
-           setPage("Login to your PZH", "login.html");
+            setPage("Login to your PZH", "login.html");
```
indentation is off

habibvirji commented on webinos/test/client/login.html:
```diff
@@ -25,8 +27,9 @@
 
     function displayLogin(html_input) {
-      $('#main').html('<article class="module width_full"><header><h3> Login to your personal zone hub </h3></header><div class="module_content">'+ html_input.payload.message +
+      $('#main').html('<article class="module width_full"><header><h3> Login to Your PZH </h3></header><div class="module_content">'+ html_input.payload.message +
```
your personal zone hub looks correct

habibvirji commented:
@kwlodarska can you please resubmit as the newer pull request requires updates to your branch
